### PR TITLE
[WIP] add `FLStatsHandler`

### PR DESCRIPTION
**Pull request by KumoLiu**
Fixes # .

### Description
add `FLStatsHandler` for NVFlare.
It mainly call `_add` in `AnalyticsSender`.
https://github.com/NVIDIA/NVFlare/blob/b0de532fc2f65de30eb8af728561bab33fdaf076/nvflare/app_common/widgets/streaming.py#L97

Since `AnalyticsSender` can only send to `FLStatsHandler` after instantiated, I add it to ignite.engine.state as a workaround.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by KumoLiu** [comment]
Hi @Nic-Ma @wyli @holgerroth @YuanTingHsieh, could you please help review it first. Not sure if this workaround is acceptable, if so then I could add unittest.

I have tested it locally with [this example](https://github.com/NVIDIA/NVFlare/tree/dev/integration/monai/examples/spleen_ct_segmentation_real-world) in NVFlare repo.

Thanks in advance!

File: docs/source/handlers.rst [hunk 1]
```diff
@@ -143,6 +143,12 @@ Tensorboard handlers
     :members:
 
 
+NVFlare stats handlers
```
Comment (Nic-Ma) [inline docs/source/handlers.rst:146]: handlers -> handler

Comment (KumoLiu) [reply docs/source/handlers.rst]: Thanks! Fixed!

File: monai/handlers/nvflare_stats_handler.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        """
+        self._sender = engine.state.extra.get(ExtraItems.STATS_SENDER, self._sender)
+        self._default_iteration_sender(engine, self._sender)
+
+    def _send_stats(
+        self, _engine: Engine, sender, tag: str, value: Any, data_type: AnalyticsDataType, step: int
+    ) -> None:
+        """
+        Write scale value into TensorBoard.
+        Default to call `Summarysender.add_scalar()`.
```
Comment (YuanTingHsieh) [inline monai/handlers/nvflare_stats_handler.py:154]: What is a `Summarysender`?

File: monai/handlers/nvflare_stats_handler.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    Engine, _ = optional_import(
+        "ignite.engine", IgniteInfo.OPT_IMPORT_VERSION, min_version, "Engine", as_type="decorator"
+    )
+
+DEFAULT_TAG = "Loss"
+
+
+class FLStatsHandler:
+    """
+    FLStatsHandler defines a set of Ignite Event-handlers for all the NVFlare ``AnalyticsSender`` logics.
```
Comment (YuanTingHsieh) [inline monai/handlers/nvflare_stats_handler.py:40]: Just from a naming point of view, do we want this to be called `FLStatsHandler` or `NVFlareStatsHandler`?

File: monai/handlers/nvflare_stats_handler.py (hunk 2 above)
Comment (KumoLiu) [reply monai/handlers/nvflare_stats_handler.py:154]: Good catch, a typo here, has fixed.

**Comment by KumoLiu** [comment]
Update: Waiting for @chesterxgchen and @holgerroth to discuss a proposal.

**Comment by KumoLiu** [comment]
The changes in this PR have been included in https://github.com/NVIDIA/NVFlare/pull/1925.
Closed for now.

**Pull request closed without merging**